Gruntfile.js

```diff
@@ -172,15 +171,11 @@ module.exports = function( grunt ) {
 				singleRun: true
 			},
 			main: {
-
-				// The Chrome sandbox doesn't work on Travis.
-				browsers: [ isTravis ? "ChromeHeadlessNoSandbox" : "ChromeHeadless" ]
+				browsers: [ "ChromeHeadless", "FirefoxHeadless" ]
```

Comment:
Does the sandboxed version work in Travis now?

Reply:
It does! You can see that Travis for this PR passed. :)

Reply:
🤔 🤣